docs/meshcloud.policies.md

```diff
@@ -75,18 +126,19 @@ Your organization is fully free to define policies across the entire meshStack.
 
 ### How are meshPolicies evaluated
 
-As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is no "direction" in a meshPolicy: the meshObjects are evaluated bi-directionally. It does not matter if you have a meshPolicy for "meshCustomer & meshProject" or "meshProject & meshCustomer", the policy evaluation is equal.
-If a policy is defined, the two meshObjects' tags will be evaluated on a so-called set intersection. This means that for either tags, at least one value must be present on both sides for a successful evaluation. If neither side have the given tag, the policy is also successful.
+As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshSubjects on their tags. There is the "direction" between the affected meshSubject and authoritative meshSubject in a meshPolicy: the meshSubjects are evaluated unidirectionally.
+If a meshPolicy is defined, the two meshSubjects' tags will be evaluated depending on the evaluation strategy. If neither side have the given tag, the meshPolicy is also successful.
 To make this more clear, the table below describes certain conditions and their behavior when policies are applied on tags.
 
-In this example, we're looking at a meshPolicy between a meshProject's `environment` tag, and a meshCustomer's `environment` tag with the following allowed values: `dev`, `qa`, and `prod`.
+In this example, we're looking at a meshPolicy between a meshProject's `environment` tag, and a meshCustomer's `environment` tag with the following allowed values: `dev`, `qa`, and `prod`. The meshProject is the affected meshSubject and the meshCustomer is the authoritative meshSubject. The chosen evaluation strategy for this example is 'Subset'.
 
 | meshProject  | meshCustomer | Result | Explanation |
 | ------------ | ------------ | ------ | ----------- |
```

Comment:
I would have expected this table at the evaluation strategy part. I think it is really helpful

Reply:
I've added another table to the evaluation strategy section. This section makes sense because here we've the explicit meshCustomer meshProject example.